src/parsers/mimic.py

```diff
@@ -0,0 +1,233 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import os
+
+from src.utils.utils import AnalysisUtils, change_col_to_datetime
+from src.utils.constants import LAB_MAPPING, LAB_VECT_COLS
+
+
+class MIMICParser(AnalysisUtils):
+    def __init__(self, data, res, gender="MF", age_b=0, age_a=100, ethnicity="WHITE", lab_mapping=None):
+        self.m_p_df = None
+        AnalysisUtils.__init__(self, data, res, gender=gender, age_b=age_b, age_a=age_a, ethnicity=ethnicity, lab_mapping=LAB_MAPPING)
```

Comment:
Added `lab_mapping` as a param so it can passed dynamically to the class instead of making change to the source code, ie, to `constants.py`.
This makes it easier to run the code using notebooks.

Reply:
Ok, but anyhow in row #12 you need to change  `LAB_MAPPING `to `lab_mapping` ?